webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/reports/ExecutionBatchTest.scala

```diff
@@ -1232,10 +1232,10 @@ class ExecutionBatchTest extends Specification {
       withBad.componentValues.size === 4
     }
     "with bad reports return a component with the key values foo with one unexpected and one repaired " in {
-      withBad.componentValues("foo").flatMap(_.messages.map(_.reportType)) === Unexpected :: EnforceRepaired :: Nil
+      withBad.componentValues("foo").flatMap(_.messages.map(_.reportType)) === EnforceRepaired :: Unexpected :: Nil
```

Comment:
not sure why the order changed, but order seems meaningless in that test

Reply:
I think it changed because the list of component values is built using two group by instead of one previously and the order might be different (but it's not important)